### DISCUSSION TITLE
Placing the hot block to ignite the fuel

### DISCUSSION THREAD

**Discussion by PurnaPushkalaSomasundaram** [firemodels/fds#13072] [Q&A]
I have a doubt regarding how to place a hot block to ignite the liquid fuel as the boiling point of the liquid is not sufficient to ignite using external thermal fluxes?

**Comment by rmcdermo** [comment]
Unless you otherwise specify an AUTO_IGNITION_TEMPERATURE, any of the fuel that evaporates will burn subject to the EXTINCTION_MODEL.  To increase the rate of evaporation for numerical purposes, we sometimes add an EXTNERNAL_FLUX to the surface for a short period of time (use a RAMP to turn it off).  See the [Pool Fires](https://github.com/firemodels/fds/tree/master/Validation/Pool_Fires/FDS_Input_Files) examples.

**Comment by PurnaPushkalaSomasundaram** [comment]
<img width="176" alt="image" src="https://github.com/firemodels/fds/assets/166021559/a386732f-75fa-4203-9f81-605d0aefbd9c">
I am uploading an image. I need to code the ignitor hot block at the temperature of 1300 C for 100 s and remove it, once the fire is ignited.

**Comment by rmcdermo** [comment]
FDS User Guide, 18.4.1 Creating and Removing Obstructions
